Neos.Media.Browser/Classes/Controller/AddTranslatedFlashMessageTrait.php

```diff
@@ -0,0 +1,40 @@
+<?php
+declare(strict_types=1);
+
+namespace Neos\Media\Browser\Controller;
+
+/*
+ * This file is part of the Neos.Media.Browser package.
+ *
+ * (c) Contributors of the Neos Project - www.neos.io
+ *
+ * This package is Open Source Software. For the full copyright and license
+ * information, please view the LICENSE file which was distributed with this
+ * source code.
+ */
+
+use Neos\Error\Messages\Message;
+use Neos\Flow\Annotations as Flow;
+
+/**
+ * A trait to add backend translation based on the backend users settings
+ */
+trait AddTranslatedFlashMessageTrait
+{
+    /**
+     * @Flow\Inject
+     * @var \Neos\Flow\I18n\Translator
+     */
+    protected $_translator;
+
+    public function addTranslatedFlashMessage(
+        string $translationIdForTheMessageBody,
+        string $severity = Message::SEVERITY_OK,
+        array $messageArguments = [],
+        ?int $messageCode = null
+    ): void {
```

Comment:
```php
public function addTranslatedFlashMessage(string $translationIdForTheMessageBody, string $severity = Message::SEVERITY_OK, array $messageArguments = [], ?int $messageCode = null): void
    {
// Your Code
```

I would combine everything in one line. That looks better.

Reply:
yes while we could argue about this i think this is fine as is. (Allowed codestyle in PSR2 and PSR12 and our code style check doesnt complain)